packages/server/test/e2e/5_spec_isolation_spec.coffee

```diff
@@ -126,7 +126,7 @@ expectRunsToHaveCorrectStats = (runs = []) ->
         test,
         "wallClockDuration",
         timings,
-        timings + 50, ## add 50ms to account for padding
+        timings + 80, ## add 80ms to account for padding
```

Comment:
should we worry about why padding used to be ok at 50ms but is now 80ms? or has this always been flaky

Reply:
hmm, let me see

Reply:
i've noticed this tends to happen much more frequently in Firefox

Reply:
do you think it could be worth looking at the firefox-specific video recording setup code/firefoxGcInterval to see if it could be bleeding into the user's test time somehow?

Reply:
I think firefox is just slower, so where chrome needed only 50ms padding, firefox needs more. It's not drastically more, but is barely on the edge